### PR TITLE
Initial project structure refactor

### PR DESCRIPTION
**Pull request by chrisfenos**
- template project structure

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+const express = require("express");
+const ethers = require('ethers');
+const app = express();
+const cors = require("cors");
+const utils = ethers.utils;
+
+const provider = new ethers.providers.JsonRpcProvider('http://40.114.122.81:8545')
+var faucetWallet = new ethers.Wallet('0x938c57c1b1e3c43407e86e38dddd4f4a92e90fb0c1d55a21c8c66e907cc84a2a');
```
Comment (5chdn) [inline server/index.js]: Private key on GitHub is not a good idea :innocent:

File: gitter/gitter-stream.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  return yesterday;
+}
+
+getAllMessages = async (userId) => {
+  const yesterday = getYesterday();  
+  await getMessagesRequest(userId, yesterday);
+}
+
+getMessagesRequest = async (userId, yesterday) => {
+  const url = `https://api.gitter.im/v1/rooms/5b92ffefd73408ce4fa74cbf/chatMessages?access_token=49a37e04d62ff8583f7c31e8eea9c55f2e0ed3da`;
```
Comment (5chdn) [inline gitter/gitter-stream.js:52]: access token on Github is not a good idea

File: gitter/gitter-stream.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
+var https = require('https');
+const axios = require('axios');
+
+var roomId    = '5b92ffefd73408ce4fa74cbf'
+var token     = '49a37e04d62ff8583f7c31e8eea9c55f2e0ed3da';
```
Comment (5chdn) [inline gitter/gitter-stream.js:5]: Is that an access token?

**Review by 5chdn — DISMISSED** [review]
Please find a way to deal with secrets first

File: gitter/gitter-stream.js (hunk 2 above)
Comment (chrisfenos) [reply gitter/gitter-stream.js:52]: fixed

File: gitter/gitter-stream.js (hunk 3 above)
Comment (chrisfenos) [reply gitter/gitter-stream.js:5]: fixed

File: server/index.js (hunk 1 above)
Comment (chrisfenos) [reply server/index.js]: fixed

**Review by ChainSafeSystems — APPROVED** [review]

**Review by ChainSafeSystems — APPROVED** [review]

**Pull request merged**